example/index.js

```diff
@@ -5,6 +5,7 @@ const config = {
     "yamcsDictionaryEndpoint": "http://localhost:9000/yamcs-proxy/",
     "yamcsHistoricalEndpoint": "http://localhost:9000/yamcs-proxy/",
     "yamcsRealtimeEndpoint": "ws://localhost:9000/yamcs-proxy-ws/",
+    "yamcsWebsocketEndpoint": "ws://localhost:9000/yamcs-proxy-ws-v2/",
```

Comment:
just curious, why do we need 2 different configs?

Reply:
I have no idea. @akhenry ?

Reply:
Because, for now, we have to use two different websocket endpoints. [There is an issue to address this](https://github.com/akhenry/openmct-yamcs/issues/40).